### ISSUE TITLE
#1042413006------------- Elements must have sufficient color contrast

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#688]
<strong>Source:</strong>`<a href="https://mcstest248.esri.com/portal/home/groups.html?sortOrder=asc&amp;sortField=title" target="_blank" rel="noopener noreferrer" class="manage-groups">`
<strong>Element Location: </strong>.manage-groups
<strong>Impact: </strong>serious
<strong>failureSummary:</strong> Fix any of the following:
  Element has insufficient color contrast of 3.05 (foreground color: #1890ff, background color: #f8f8f8, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
Tags: cat.color,wcag2aa,wcag143,ACT
Screenshot:`C:\Users\khy12817\Desktop\Test.PNG`
